test/unit_tests/braket/experimental/autoqasm/test_parameters.py

```diff
@@ -629,22 +683,22 @@ def main(basis: int):
             x(0)
         c = measure(0)  # noqa: F841
 
-    measurements = _test_parametric_on_local_sim(main(FreeParameter("basis")), {"basis": 0})
+    measurements = _test_parametric_on_local_sim(main, {"basis": 0})
     assert all(val == 1 for val in measurements["c"])
-    measurements = _test_parametric_on_local_sim(main(FreeParameter("basis")), {"basis": 1})
+    measurements = _test_parametric_on_local_sim(main, {"basis": 1})
     assert 1 in measurements["c"] and 0 in measurements["c"]
 
 
 def test_sim_comparison_stmts():
     @aq.main
-    def main(basis: int):
+    def main(basis: float):
         if basis > 0.5:
             x(0)
```

Comment:
Basis seems like a weird name for a float, my bad for writing it like that. Mind updating this for me? No worries if you want to skip, it's definitely out of scope 😝 
```suggestion
    def main(threshold: float):
        if threshold > 0.5:
            x(0)
```

Reply:
Sure! Definitely got a laugh when this test started failing and I realized that as written, with basis as an int, this should have been failing the whole time and the new failure was an improvement lol

Reply:
bump